### PR TITLE
add clarification for csr

### PR DESCRIPTION
**Pull request by eric-haibin-lin**

### CODE REVIEW

**Comment by eric-haibin-lin** [comment]
@cjolivier01 @reminisce

File: python/mxnet/ndarray/sparse.py [hunk 1]
```diff
@@ -247,6 +247,9 @@ class CSRNDArray(BaseSparseNDArray):
     row i are stored in indices[indptr[i]:indptr[i+1]] and their corresponding values are stored
     in values[indptr[i]:indptr[i+1]].
 
+    The indices are expected to be sorted per row in ascending order. Duplicate matrix entries are
```
Comment (cjolivier01) [inline python/mxnet/ndarray/sparse.py]: should it be 'duplicate column entries for the same row'?

Comment (cjolivier01) [inline python/mxnet/ndarray/sparse.py]: nit: "The column indices for a given row are expected to be in ascending order..."

Comment (eric-haibin-lin) [inline python/mxnet/ndarray/sparse.py]: Done

**Pull request merged**